### DISCUSSION TITLE
enable to Copy-PnPFile to a channel with spaces in the name.

### DISCUSSION THREAD

**Discussion by RichardVanDriel** [pnp/powershell#3863] [Q&A]
$TargetUrlVK = "/sites/VKKT-Trajecten-" + $Channelname + "/Shared Documents"
    Copy-PnPFile -SourceUrl "Shared Documents/TOxxxxxx" -TargetUrl "$TargetUrlVK" #-Overwrite

Hi, if I want to copy files/folders to an channel ($TargetUrlVK) with spaces in the name i get a not found error.
If I do the same to a channel without spaces everysting works fine.

Anyone an idea why and how to fix?

TIA
Richard

**Comment by Barbarur** [comment]
I would look at the PATH of the destination channel. Make sure there is not double space, or maybe both words got combined. That's the most common issue with spaces.
